src/v2/params/index.ts

```diff
@@ -20,10 +20,6 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-/**
- * @hidden
- * @alpha
- */
```

Comment:
q: What's going on here?

Reply:
Making these fields documented + visible in code complete.

Let's hold back on this until we're ready for the final release.

Reply:
Done.